js/components/PortfolioHeader.jsx

```diff
@@ -10,14 +10,28 @@ var PortfolioHeader = React.createClass({
 
   render: function() {
     var repsAvailable = '';
+    var dividends = '';
     if (this.props.reps > -1) {
-      repsAvailable = <p>{strings.PORTFOLIO_HEADER_REPS(this.props.reps)}
-        <span className="reps-available-info glyphicon glyphicon-info-sign" data-toggle="popover" data-placement="right" data-content={strings.REPS_AVAILABLE_INFO_CONTENT}></span></p>;
+      repsAvailable = (
+        <div>{strings.PORTFOLIO_HEADER_REPS(this.props.reps)}
+        <span className="reps-available-info glyphicon glyphicon-info-sign"
+          data-toggle="popover" data-placement="right"
+          data-content={strings.REPS_AVAILABLE_INFO_CONTENT}></span></div>
+      )
+    }
+
+    if (this.props.dividends) {
```

Comment:
If a user had no investments, then they wouldn't be an investor for the category. But, the check isn't hurting anybody.